hummingbot/client/settings.py

```diff
@@ -201,13 +262,7 @@ def ethereum_required_trading_pairs() -> List[str]:
 MAXIMUM_TRADE_FILLS_DISPLAY_OUTPUT = 100
 
 
-CONNECTOR_SETTINGS = _create_connector_settings()
-DERIVATIVES = {cs.name for cs in CONNECTOR_SETTINGS.values() if cs.type is ConnectorType.Derivative}
-EXCHANGES = {cs.name for cs in CONNECTOR_SETTINGS.values() if cs.type is ConnectorType.Exchange}
-OTHER_CONNECTORS = {cs.name for cs in CONNECTOR_SETTINGS.values() if cs.type is ConnectorType.Connector}
-ETH_WALLET_CONNECTORS = {cs.name for cs in CONNECTOR_SETTINGS.values() if cs.use_ethereum_wallet}
-ALL_CONNECTORS = {"exchange": EXCHANGES, "connector": OTHER_CONNECTORS, "derivative": DERIVATIVES}
-EXAMPLE_PAIRS = {name: cs.example_pair for name, cs in CONNECTOR_SETTINGS.items()}
-EXAMPLE_ASSETS = {name: cs.example_pair.split("-")[0] for name, cs in CONNECTOR_SETTINGS.items()}
+EXAMPLE_PAIRS = {name: cs.example_pair for name, cs in AllConnectorSettings.get_connector_settings().items()}
```

Comment:
We don't need to keep these 2 constants (they can be class method like all the others). Let's keep it clean, keep constants for only primitive values.